src/Command/BakeSimpleMigrationCommand.php

```diff
@@ -188,6 +188,10 @@ public function buildOptionParser(ConsoleOptionParser $parser): ConsoleOptionPar
             'short' => 'f',
             'boolean' => true,
             'help' => 'Force overwriting existing file if a migration already exists with the same name.',
+        ])->addOption('source', [
+            'short' => 's',
+            'default' => 'Migrations',
+            'help' => 'Name of the folder in which the migration should be saved.',
```

Comment:
Not sure if we have a coding standard for that. But I like to mention what a given option defaults to.

```suggestion
            'help' => 'Name of the folder in which the migration should be saved.' Defaults to "Migrations".,
```

Reply:
Apparently the default is already mentioned Automatically after the Text

Reply:
Yes, the default value will be automatically attached at the end of the Text. In this case it would be

> Name of the folder in which the migration should be saved. (default: Migrations)